socorro/external/postgresql/models.py

```diff
@@ -1266,19 +1550,30 @@ class SpecialProductPlatform(DeclarativeBase):
 class TcbsBuild(DeclarativeBase):
     __tablename__ = 'tcbs_build'
 
-    #column definitions
-    build_date = Column(u'build_date', DATE(), primary_key=True, nullable=False)
-    hang_count = Column(u'hang_count', INTEGER(), nullable=False, server_default=text('0'))
-    lin_count = Column(u'lin_count', INTEGER(), nullable=False, server_default=text('0'))
-    mac_count = Column(u'mac_count', INTEGER(), nullable=False, server_default=text('0'))
-    process_type = Column(u'process_type', CITEXT(), primary_key=True, nullable=False)
-    product_version_id = Column(u'product_version_id', INTEGER(), primary_key=True, nullable=False, autoincrement=False)
-    release_channel = Column(u'release_channel', CITEXT(), nullable=False)  # DEPRECATED
-    report_count = Column(u'report_count', INTEGER(), nullable=False, server_default=text('0'))
-    report_date = Column(u'report_date', DATE(), primary_key=True, nullable=False)
-    signature_id = Column(u'signature_id', INTEGER(), primary_key=True, nullable=False)
+    # column definitions
+    build_date = Column(u'build_date', DATE(),
+                        primary_key=True, nullable=False)
+    hang_count = Column(u'hang_count', INTEGER(),
+                        nullable=False, server_default=text('0'))
+    lin_count = Column(u'lin_count', INTEGER(),
+                       nullable=False, server_default=text('0'))
+    mac_count = Column(u'mac_count', INTEGER(),
+                       nullable=False, server_default=text('0'))
+    process_type = Column(u'process_type', CITEXT(),
+                          primary_key=True, nullable=False)
+    product_version_id = Column(u'product_version_id', INTEGER(
+    ), primary_key=True, nullable=False, autoincrement=False)
+    release_channel = Column(u'release_channel', CITEXT(),
+                             nullable=False)  # DEPRECATED
+    report_count = Column(u'report_count', INTEGER(),
+                          nullable=False, server_default=text('0'))
+    report_date = Column(u'report_date', DATE(),
+                         primary_key=True, nullable=False)
+    signature_id = Column(u'signature_id', INTEGER(),
+                          primary_key=True, nullable=False)
```

Comment:
Most of these lines are under 100 characters wide, so they're actually all fine. When you run autopep8 going forward, please use ``-max-line-length 100``.